modules/components/pages/outputs/schema_registry.adoc

```diff
@@ -285,9 +274,136 @@ password: ${KEY_PASSWORD}
 
 The maximum number of messages to have in flight at a given time. Increase this number to improve throughput.
 
-
 *Type*: `int`
 
 *Default*: `64`
 
+=== `oauth`
```

Comment:
Please check these field descriptions as I have reworded slightly to make them less generic.

Reply:
Same

Reply:
Added a ticket for this: https://github.com/redpanda-data/documentation-private/issues/2760